eval.py

```diff
@@ -218,7 +228,8 @@ def decisions(agent, data, window_size, debug, stock, api):
                 file.write(str(datetime.datetime.now().strftime("%m/%d/%Y,%H:%M:%S")) + ',SELL,$' + str(
                     date.get(stock)[0].c) + '\n')
                 file.close()
-                submit_order_helper(1, stock, 'sell', api)
+                position = api.get_position(stock)
```

Comment:
Alpaca is a little bit glitchy when it comes to selling all the stocks, I know for sure if I were to run this it would fail and wouldn't sell any. Maybe try caching your last buy qty and sell that amount?

Reply:
Sorry for being AFK so long, but I can answer this now. what I did was 
`try:
       submit_order_helper(1, stock, 'sell', API)
except:
            print('failsafe activated...')`

Reply:
> Alpaca is a little bit glitchy when it comes to selling all the stocks, I know for sure if I were to run this it would fail and wouldn't sell any. Maybe try caching your last buy qty and sell that amount?

Actually, I have been running this perfectly for the past 1 week